Assets/MRTK/Core/Inspectors/Utilities/InspectorUIUtility.cs

```diff
@@ -6,6 +6,7 @@
 using System.Linq;
 using System.Reflection;
 using UnityEditor;
+using UnityEditor.Experimental.SceneManagement;
```

Comment:
why do we need to reference scenemanagement here? Seems unrelated.

Reply:
it's for the PrefabStageUtility to do the prefab checks for nested scriptable assets.